src/views/AutocompleteComponent.tsx

```diff
@@ -1,21 +1,25 @@
 import * as React from "react";
 import {Suggestion} from "../plugins/autocompletion_utils/Common";
 import * as css from "./css/main";
+import {HighlightSequencedSuggestion} from '../SequenceKit'
 
 interface SuggestionProps {
     suggestion: Suggestion;
     onHover: () => void;
     onClick: () => void;
     isHighlighted: boolean;
+    searchKey: string;
 }
 
-const SuggestionComponent = ({suggestion, onHover, onClick, isHighlighted}: SuggestionProps) =>
+const SuggestionComponent = ({suggestion, onHover, onClick, isHighlighted, searchKey}: SuggestionProps) =>
     <li style={css.autocomplete.item(isHighlighted)}
         onMouseOver={onHover}
         onClick={onClick}>
 
         <i style={Object.assign({}, css.suggestionIcon, suggestion.style.css)} dangerouslySetInnerHTML={{__html: suggestion.style.value}}/>
-        <span style={css.autocomplete.value}>{suggestion.displayValue}</span>
+
+        <span style={css.autocomplete.value}
+              dangerouslySetInnerHTML={{__html: HighlightSequencedSuggestion( suggestion.displayValue, searchKey)}} />
```

Comment:
I thought that is better for speed reasons. But okay I'll make it react component based.